CodeMaid/UI/Dialogs/About/AboutWindow.xaml

```diff
@@ -1,15 +1,15 @@
-<platformUI:DialogWindow x:Class="SteveCadwallader.CodeMaid.UI.Dialogs.About.AboutWindow"
+<platformUi:DialogWindow x:Class="SteveCadwallader.CodeMaid.UI.Dialogs.About.AboutWindow"

```

Comment:
Capitalization could be reset to "platformUI" for consistency with other windows.